lib/apiservers/engine/backends/container.go

```diff
@@ -985,22 +984,25 @@ func (c *Container) ContainerInspect(name string, size bool, version version.Ver
 	}
 	var started time.Time
 	var stopped time.Time
-	if results.Payload.ProcessConfig.StartTime != nil && *results.Payload.ProcessConfig.StartTime > 0 {
```

Comment:
@fdawg4l There are a few cosmetic changes, but I would gloss over them and focus on the pointer issue.  Vast majority of the changes I've seen so far are due to Swagger and not cosmetic.